lago/export.py

```diff
@@ -203,20 +215,31 @@ def export(self):
                 self.sparse()
                 self.rebase()
                 self.calc_sha('sha1')
-                self.calc_sha('sha512')
                 self.update_lago_metadata()
                 self.write_lago_metadata()
                 self.compress()
                 rollback.clear()
 
 
 class FileExportManager(DiskExportManager):
+    """
+    FileExportManager is responsible exporting images of type file and empty.
+
+    Attributes:
+        standalone (bool): If true, create a new image which is the result of
+         merging all the layers of src (the image that we want to export).
+         src_qemu_info (dict): Metadata on src which was generated by qemu-img.
```

Comment:
I think you need to keep the same indentation, otherwise it would not format nicely with sphinix:
```python
"""
Attributes:
    standalone (bool): long line ...
        more of that long line
    othervar (str): blah
"""
```